octodns/provider/ns1.py

```diff
@@ -233,11 +233,11 @@ def _params_for_A(self, record):
                         },
                     )
             params['filters'] = []
-            if len(params['answers']) > 1:
-                params['filters'].append(
-                    {"filter": "shuffle", "config": {}}
-                )
             if has_country:
+                if len(params['answers']) > 1:
```

Comment:
Looks like this change probably needs a corresponding tweak in the tests as this `if` always evaluates to true.

Reply:
Would removing it still apply a "shuffle" filter to records with a single answer? Not sure that'd be a problem, but I assume that was the original intent behind the `if > 1`